Remotion/Web/Development.WebTesting/WebTestException.cs

```diff
@@ -25,6 +25,11 @@ public WebTestException (string message)
     {
     }
 
+    public WebTestException (string message, string browser, string browserVersion, string driverVersion)

```

Comment:
Since we don't use the ctors directly, we should move the driver-overload into the factory.